### DISCUSSION TITLE
mp3/bluetooth

### DISCUSSION THREAD

**Discussion by FinnHansen** [pschatzmann/ESP32-A2DP#519] [Q&A] [answered]
Hi.

Yes im a newbee, but anyway.....
I looking for a way to create a mp3 device, there plays only one mp3 file, when you hit a button.
I have tried to build from /player-sdfat-a2dp.
Any help would be nice,

**Comment by XPMonline** [comment, marked as answer]
Hi, you should look for sime audio libraries, such as audio-tools, or "audioI2s". ESP32 can be connected to the SD card reader and you can play your songs from it.
